extensions/azurecore/src/azureResource/services/terminalService.ts

```diff
@@ -174,8 +174,13 @@ class AzureTerminal implements vscode.Pseudoterminal {
 		}
 	}
 
+	private areSameDimensions(oldDimensions: vscode.TerminalDimensions, newDimensions: vscode.TerminalDimensions): boolean {
+		return (oldDimensions && (oldDimensions.columns === newDimensions.columns &&
```

Comment:
Could simplify a bit 

`oldDimensions?.columns === newDimensions.columns && oldDimensions?.rows === newDimensions.rows`

Also the oldDimensions param should be typed as possibly undefined- `oldDimensions: vscode.TerminalDimensions | undefined`